recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Thanks! Can you xref that in your next PRs. (I see you are sending a few of those now.)